www/.eslintrc.json

```diff
@@ -1,5 +1,19 @@
 {
   "globals": {
-    "graphql": true
+    "graphql": false
```

Comment:
why was this changed?

Reply:
this was a mistake changing it back now.